### PR TITLE
Mouse support

### PR DESCRIPTION
**Pull request by alvarogonzalezsotillo**
I added some mouse support:

- Right click shows a popup menu with the tab groups
- Mouse wheel invokes `centaur-tabs-backward/forward`

### CODE REVIEW

**Comment by ema2159** [comment]
Awesome! Let me test it and check if everything is alright to merge! Thanks!

File: centaur-tabs.el [hunk 1]
```diff
@@ -1051,6 +1051,15 @@ hidden, it is shown again.  Signal an error if Centaur-Tabs mode is off."
 (defvar centaur-tabs-mode-map
   (let ((km (make-sparse-keymap)))
     (define-key km centaur-tabs-prefix-key centaur-tabs-prefix-map)
+
+    ;;; Use mouse wheel to switch between buffers of same group
+    (define-key km (kbd "<header-line> <mouse-4>") 'centaur-tabs-forward )
```
Comment (alvarogonzalezsotillo) [inline centaur-tabs.el]: It seems that I changed the "natural" behaviour of the mouse wheel: mouse wheel up should be a tabs-backwards but I implemented a tabs-forward.

Do you like to make this change yourself? Or do you prefer me to make another pull request?

**Comment by ema2159** [comment]
By the way, awesome PR!

File: centaur-tabs.el (hunk 1 above)
Comment (alvarogonzalezsotillo) [reply centaur-tabs.el]: Done!
